test/Grpc.Net.Client.Tests/GrpcChannelTests.cs

```diff
@@ -148,18 +150,37 @@ public async Task Build_HttpHandler_UsedForRequestsAsync()
             Assert.AreEqual("HttpHandler", ex.Status.DebugException.Message);
         }
 
+#if NET472
+        [Test]
+        public void Build_NoHttpProviderOnNetFx_Throw()
+        {
+            // Arrange & Act
+            var ex = Assert.Throws<PlatformNotSupportedException>(() => GrpcChannel.ForAddress("https://localhost"));
+
+            // Assert
+            var message =
+                $"gRPC requires extra configuration to successfully make RPC calls on older platforms such " +
+                $"as .NET Framework. An HTTP provider must be specified using {nameof(GrpcChannelOptions)}.{nameof(GrpcChannelOptions.HttpHandler)} or " +
+                $"{nameof(GrpcChannelOptions)}.{nameof(GrpcChannelOptions.HttpClient)}. The configured HTTP provider must either support HTTP/2 or " +
+                $"be configured to use gRPC-Web. See https://aka.ms/pzkMXDs for details.";
```

Comment:
nit: the link points here: https://docs.microsoft.com/en-us/aspnet/core/grpc/basics?view=aspnetcore-5.0 is that indended?

Reply:
Its a temporary link until docs are added. That will happen when this feature is released.

Reminder issue - https://github.com/grpc/grpc-dotnet/issues/1200